homeworks/nikita.karchinskiy_Nikkarchinskiy/6-typescript/src/index.ts

```diff
@@ -0,0 +1,69 @@
+const letterBox = document.querySelector<HTMLDivElement>('[data-letter-task]')!;
+const scoreBox = document.querySelector<HTMLDivElement>('[data-score]')!;
+const scoreDif = document.querySelector<HTMLDivElement>('[data-circle]')!;
+const mainWrapper = document.querySelector<HTMLDivElement>('[data-main-wrapper]')!;
+
+let score: number = 100;
+let letter: string;
+let timer: number;
+scoreBox.innerText = '100';
+
+function getRandom(min: number, max: number): number {
+    min = Math.ceil(min);
+    max = Math.floor(max);
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+function lostCase(): void {
+    scoreBox.innerText = 'Lost';
+    letterBox.innerText = '';
+    mainWrapper.classList.add('main-wrapper__lost');
+}
+
+function winCase(): void {
+    scoreBox.innerText = 'WIN';
+    letterBox.innerText = '';
+    mainWrapper.classList.add('main-wrapper__win');
+}
+
+function showLetter(): void {
+    letter = String.fromCharCode(getRandom(65, 90));
+    letterBox.innerText = letter;
+    timer = window.setTimeout(() => {
+        let randomEmpty: number;
+        randomEmpty = getRandom(10, 15);
```

Comment:
This variable can be a const because you never reassign it. So please, make it `const ` and save the value in the variable directly when you initialize it.